content/languages/javascript/mocha.md

```diff
@@ -3,6 +3,36 @@ title: Mocha
 tags: [javascript, reference, testing]
 ---
 
+# Mocha
+
+## Basic Setup
+
+### Solution
+
+```javascript
+const add = (a, b) => a + b;
+```
+
+### Tests
+
+```javascript
+const {assert, config, expect} = require("chai");
+
+// don't truncate assertion diff output
+config.truncateThreshold = 0;
+
+// if applicable to the challenge, don't truncate printing deeply-nested objects
+require("util").inspect.defaultOptions.depth = null;
+
+describe("add", () => {
+  it("should add two numbers", () => {
+    assert.equal(add(1, 2), 3);
```

Comment:
`assert.strictEqual` should be used.

Reply:
You're right, and I'll have to change that at Q as well. Big oversight!